src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/mapper/induction/PreviousWorkExperiencesResourceMapper.kt

```diff
@@ -22,7 +23,7 @@ import java.time.ZoneOffset
 import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.induction.WorkExperience as WorkExperienceDomain
 import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.WorkExperience as WorkExperienceApi
 
-@Mapper
+@Mapper(nullValueIterableMappingStrategy = NullValueMappingStrategy.RETURN_DEFAULT)
```

Comment:
This `Iterable` mapping strategy ensures that null collections are always mapped to empty ones, thereby avoiding NPEs. However, because we wish to keep in sync with the existing CIAG API, this isn't desirable for response objects, hence the use of `.ifEmpty { null }` below.